casepro/cases/models.py

```diff
@@ -103,6 +103,16 @@ def __str__(self):
         return self.name
 
 
+class SystemUser(User):
+
+    @classmethod
+    def get_or_create(cls):
+        systemUsers = cls.objects.filter()
+        if len(systemUsers) > 0:
+            return systemUsers[0]
```

Comment:
Ah, I thought because it's a separate inherited class (`SystemUser` inherits from `User`), django creates a separate `cases_systemuser` table.